src/pages/login/styles/login.css

```
@@ -0,0 +1,120 @@
+  position: relative;
+  display: flex;
+  align-items: center;
+}
+
+.sf-eye {
+  position: absolute;
+  right: 8px;
+  top: 50%;
+  transform: translateY(-50%);
+  
+  border: none;
```

Stylelint (17.5.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)